File: 1 basic/cpp/1006. 换个格式输出整数 (15).cpp
#include<iostream> 
#include<string>
using namespace std;

int main(){
	int n;
	scanf("%d",&n);
	int b = n / 100;
	int s = n % 100 / 10;
	int g = n % 10;
	string ans;
	for(int i=0; i<b; ++i) ans += 'B';
	for(int i=0; i<s; ++i) ans += 'S';
	for(int i=1; i<=g; ++i) ans += char(i + '0');
	cout<<ans<<endl;
		
	return 0;
}

File: 1 basic/py/1006. 换个格式输出整数 (15).py
n = int(input())
b = n / 100
s = n % 100 / 10
g = n % 10
ans = ''
for i in range(int(b)): ans = ans + 'B'
for i in range(int(s)): ans = ans + 'S'
for i in range(1,int(g)+1): ans = ans + str(i)
print(ans)